main/lsp/LSPTypechecker.cc

```diff
@@ -321,7 +321,8 @@ vector<core::FileRef> LSPTypechecker::runFastPath(LSPFileUpdates &updates, Worke
             continue;
         }
 
-        absl::c_set_intersection(changedFieldHashes, oldHash.usages.symbols, std::back_inserter(intersection));
+        // TODO(jez) Just have "changedSymbolHashes" and be done with it.
```

Comment:
I guess you are going to fix this in a followup PR?

Reply:
working on it right now.